MatrixSDK/VoIP/MXCallManager.m

```diff
@@ -1267,6 +1377,200 @@ - (void)placeCallAgainst:(NSString *)phoneNumber
     } failure:failure];
 }
 
+#pragma mark - Virtual Rooms
+
+- (void)setVirtualRoomsSupported:(BOOL)virtualRoomsSupported
+{
+    _virtualRoomsSupported = virtualRoomsSupported;
+    
+    [[NSNotificationCenter defaultCenter] postNotificationName:kMXCallManagerVirtualRoomsSupportUpdated
```

Comment:
We should post it only on actual changes.

Reply:
👍